src/renderer/windows/App.tsx

```diff
@@ -31,14 +33,27 @@ export default function App() {
     const models = useService(ModelsManagerService);
     const notification = useService(NotificationService);
     const config = useService(ConfigurationService);
+    const autoUpdater = useService(AutoUpdaterService);
 
     const location = useLocation();
     const navigate = useNavigate();
 
     useEffect(() => {
+        checkIsUpdated();
         checkOneClicks();
     }, []);
 
+    const checkIsUpdated = async () => {
+        const appVersion = await lastValueFrom(autoUpdater.getAppVersion());
+        const lastAppVersion = autoUpdater.getLastAppVersion();
+
+        autoUpdater.setLastAppVersion();
```

Comment:
Update the code with previous comment, and use `appVersion` to update the lastAppVersion